adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/JsonSerializerConfig.kt

```diff
@@ -0,0 +1,12 @@
+package net.adoptopenjdk.api.v3
+
+import io.quarkus.jsonb.JsonbConfigCustomizer
+import javax.inject.Singleton
+import javax.json.bind.JsonbConfig
+
+@Singleton
+class JsonSerializerConfig : JsonbConfigCustomizer {
```

Comment:
`ContextResolver` `@Provider` wasn't used anymore, updated to recommended configuration.